.evergreen/generate.yml

```
@@ -1,10 +1,17 @@
 functions:
   run-generate-tasks:
-    command: shell.exec
+    command: subprocess.exec
     params:
       working_dir: ui
-      script: |
-        node .evergreen/scripts/generate-tasks.js
+      binary: node .evergreen/scripts/generate-tasks.js
+
+  run-generate-parallel-e2e-tasks:
+    command: subprocess.exec
+    params:
+      working_dir: ui
+      env:
+        BUILD_VARIANT: ${build_variant}
+      binary: node .evergreen/scripts/generate-parallel-e2e-tasks.js
```

[nit] As per the [docs](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands#subprocessexec), I think the preferred usage would be
```suggestion
      binary: "node"
      args:
        - ".evergreen/scripts/generate-parallel-e2e-tasks.js"
```